package.json

```diff
@@ -16,105 +16,106 @@
     "start:development:client": "node client/scripts/start.js"
   },
   "dependencies": {
-    "argon2": "^0.16.2",
-    "autoprefixer": "^7.1.2",
-    "axios": "^0.16.2",
```

Comment:
[0.17.0](https://github.com/axios/axios/blob/master/CHANGELOG.md#0170-oct-21-2017) introduced two breaking changes that may impact flood.